sbi/inference/posterior.py

```diff
@@ -64,34 +65,32 @@ def __init__(
             sample_with_mcmc: Whether to sample with MCMC. Will always be `True` for SRE
                 and SNL, but can also be set to `True` for SNPE if MCMC is preferred to
                 deal with leakage over rejection sampling.
-            mcmc_method: If MCMC sampling is used, specify the method here: either of
-                slice_np, slice, hmc, nuts.
+            mcmc_method: Method used for MCMC sampling, one of `slice_np`, `slice`, `hmc`, `nuts`.
+            mcmc_parameters: Dictionary overriding the default parameters for MCMC,
+                such as `thin` or `warumup_steps`. See signature of `_sample_posterior_mcmc`
```

Comment:
This is sending the user to the signature of a 'private' method for information. As a private method, I believe its API will not be / should not be documented publicly. On these grounds, perhaps we do want to copy the default values here, even if it is annoying.